src/framework/audio/worker/internal/workerchannelcontroller.cpp

```diff
@@ -118,8 +118,18 @@ void WorkerChannelController::init(std::shared_ptr<IWorkerPlayback> playback)
         channel()->addReceiveStream(StreamName::PlaybackDataMainStream, mainStreamId, playbackData.mainStream);
         channel()->addReceiveStream(StreamName::PlaybackDataOffStream, offStreamId, playbackData.offStream);
 
-        RetVal2<TrackId, AudioParams> ret = m_playback->addTrack(seqId, trackName, playbackData, params);
-        channel()->send(rpc::make_response(msg, RpcPacker::pack(ret)));
+        if (!soundFontRepository()->soundFonts().empty()) {
```

Comment:
This check only makes sense for tracks that will use soundfonts. Perhaps we should also check what source type is used in the params (only Fluid is important for the check)